app/renderer/components/task_card.js

```diff
@@ -92,6 +92,8 @@ module.exports = {
             .then(() => {
                 this.stop();
                 this.$emit('remove');
+            }, () => {
+                return;
```

Comment:
Do we need the reject callback if we are only returning?

Reply:
Fair enough, but in was case is the promise rejected? I haven't checked the library enough yet

Reply:
https://github.com/limonte/sweetalert2#handling-dismissals
another workaround is to use `.catch(swal.noop)`, but then I have to call the swal global variable inside the components, so I just wrote those empty callbacks

Reply:
No need, your solution seems reasonable, maybe the return may be removed, or the wrapper could be improved so it doesn't throw errors. I was confused on how it worked because rejections are usually reserved for actual errors (not an user response) and I didn't see any error when trying them without catch